apps/ledger-live-desktop/src/renderer/modals/Send/fields/AmountField.tsx

```diff
@@ -22,6 +22,7 @@ type Props = {
   initValue?: BigNumber;
   walletConnectProxy?: boolean;
   resetInitValue?: () => void;
+  withMaxLabel?: boolean;
```

Comment:
Sorry to be nitpicking on that prop name again :pray::slightly_smiling_face: but can you please change to `withUseMaxLabel`? Since they're both max, `withMaxLabel` is confusing.